### ISSUE TITLE
SlevomatCodingStandard.Variables.UnusedVariable: false positive assigning to global

### ISSUE THREAD

**Issue by jrfnl** [slevomat/coding-standard#849]
```php
function setGlobalBar() {
    global $bar;
    $bar = new FooBar();
}
```

generates a `Unused variable $bar`, while the assignment to the global is the actual functionality of the function.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/f471a2ba8bc532817cef945e01da7710f5e45ef3

**Comment by jrfnl** [comment]
@kukulich Thanks for the quick fix!
